2/utils/generate

```diff
@@ -725,8 +725,8 @@ function createTwemoji(re) {
         // used to find HTML special chars in attributes
         rescaper = /[&<>'"]/g,
 
-        // nodes with type 1 which should **not** be parsed (including lower case svg)
-        shouldntBeParsed = /IFRAME|NOFRAMES|NOSCRIPT|SCRIPT|SELECT|STYLE|TEXTAREA|[a-z]/,
+        // nodes with type 1 which should **not** be parsed
+        shouldntBeParsed = /^(iframe|noframes|noscript|script|select|style|textarea)$/,
```

Comment:
one last thing please, `^(?:iframe|.......|textarea)$` to avoid capturing on `.test`, thank you!